### ISSUE TITLE
Timer continues counting when app is minimized or screen is off

### ISSUE THREAD

**Issue by jeffpyke** [proginosko/LeechBlockNG#495]
Thank you for this useful extension.
On Firefox on Android, using leechblock NG extension, I am using the "time limit" option. I have "Count time spent on these sites only when browser tab is active" checked. The problem is if I switch to another app, the timer continues on the page.
I expect the timer to only count when the app is open and the tab is open. This is frustrating for usage because it causes my timer to frequently count down to zero when I haven't actually been browsing for that long.

**Comment by proginosko** [comment]
Yes, this is a known issue (see comments on #81 and #418). I'm currently toying with adding an option to stop the timer after N seconds of inactivity on a page to work around this problem.

**Comment by fdawans** [comment]
Hello,
Thanks for the app.
I had a similar problem as my counter was always at zero and I didn't understand the issue.
Sadly, most features are useless if the timer is not reliable...
Workaround could be to kill all open tabs on exit but that is very inconvenient.

**Comment by fdawans** [comment]
Thanks, but the timer is still going down when I am out of the browser.

**Comment by proginosko** [comment]
Follow #81 for further updates.
